crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -208,6 +209,7 @@ module bridge::bridge {
         assert!(!inner.paused, EBridgeUnavailable);
         assert!(chain_ids::is_valid_route(inner.chain_id, target_chain), EInvalidBridgeRoute);
         assert!(target_address.length() == EVM_ADDRESS_LENGTH, EInvalidEvmAddress);
+        assert!(token.balance().value() > 0, ETokenValueIsZero);
```

Comment:
can we move this after the assignment below at line 214. (`let amount = token.balance().value();`)? and can we also remove the line 218 (`let token_amount = token.balance().value();`)? 
That seems the right thing to me, not sure if it makes sense